packages/mint-components/src/components/tax-and-cash/sqm-indirect-tax-form/sqm-indirect-tax-form.feature

```diff
@@ -0,0 +1,189 @@
+@owner:andy @author:andy
+Feature: Indirect Tax Form
+
+  Background: A user has submitted their personal information in Tax Form Step One
```

Comment:
@Locrian24 @AndiLuo I think these specs just need a general review from recent changes around the businessEntity stuff